scripts/main.py

```diff
@@ -4,13 +4,23 @@
 from cryptext.session import SessionEnvironment
 from cryptext.shell import Shell
 from cryptext.command_register import Register
+from cryptext.utils.arguments import arguments
 
 
 def main():
-    Register.register(Shell, 'rw')
-    session = SessionEnvironment()
-    shell = Shell(session)
-    shell.cmdloop()
+
+    args = arguments()
+    session = SessionEnvironment(args.session)
+
+    if all(arg is None for arg in args.__dict__.values()):
+        Register.register(Shell, 'rw')
+        shell = Shell(session)
+        shell.cmdloop()
+    else:
+        if args.label[0] in list(session.content.keys()):
+            session.print_content(args.label[0], True)
+        else:
+            print('obviously need to be changed but it\'s a draft')
```

Comment:
The code above seems complicated. You know which sessions are possible and you could give these values to the parser to force that the session parameter is in the possible choices. Something like:
``` python
# in arguments
def argument(session_choices: List[str])
    ...
    parser.add_argument(
        'session',
        metavar='s',
        type=str,
        choices=session_names,
        required=False,
        default=None,
        help='store the session name',
    )

# In main
session = SessionEnvironment()
args = argument(session_names=session.content.keys())
```

Reply:
Then for the simple label option, you can simply use a default argument:
``` python
   parser.add_argument(
        '--label',
        type='str',
        required=False,
        default=None,
        help='label name',
    )
```
And in the main program, something like
``` python
     if args.label:
        session.print_content(args.label[0], True)
        return
    Register.register(Shell, 'rw')
    shell = Shell(session)
    shell.cmdloop()
```

Reply:
There may be some mistakes or inconsistencies between what I wrote and what is needed, but in principle this kind of argument management is cleaner